aspnetcore/tutorials/first-mvc-app/adding-controller.md

```diff
@@ -30,10 +30,10 @@ We cover these concepts in this tutorial series and show you how to use them to
 
 # [Visual Studio](#tab/visual-studio)
 
-* In **Solution Explorer**, right-click **Controllers > Add > Controller**
-  ![Contextual menu](adding-controller/_static/add_controller.png)
+* In the **Solution Explorer**, right-click **Controllers > Add > Controller**
+  ![Solution Explorer, right click Controllers > Add > Controller](adding-controller/_static/add_controller.png)
```

Comment:
@wadepickett this is the correct image, can you figure out why it's not rendering. It's a year old.